packages/contracts/test/registry/plugin-repo-registry.ts

```diff
@@ -1,16 +1,20 @@
 import {expect} from 'chai';
 import {ethers} from 'hardhat';
 
-import {DAO, PluginRepo} from '../../typechain';
+import {DAO, PluginRepo, ENSSubdomainRegistrar} from '../../typechain';
 import {customError} from '../test-utils/custom-error-helper';
 import {deployNewDAO} from '../test-utils/dao';
 import {deployNewPluginRepo} from '../test-utils/repo';
+import {deployENSSubdomainRegistrar} from '../test-utils/ens';
+import {ensDomainHash, ensLabelHash} from '../../utils/ensHelpers';
 
 const EVENTS = {
   PluginRepoRegistered: 'PluginRepoRegistered',
 };
 
-describe('Aragon-Plugin-Registry', function () {
+describe('PluginRepoRegistry', function () {
+  let signers:any;
```

Comment:
Let's use explicit types.

You can import `import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';` and use 
`let signers: SignerWithAddress[];`

The same goes for 
`let pluginRepoRegistry: PluginRepoRegistry;`
which you can do after `import {PluginRepoRegistry} from '../../typechain';`